libs/common-utils/src/fractionUtils.ts

```diff
@@ -169,4 +174,43 @@ export class FractionUtils {
 
     return amount.equalTo(0) ? CurrencyAmount.fromRawAmount(amount.currency, 1) : amount
   }
+
+  static simplify(fraction: Fraction): Fraction {
+    return reduce(trimZeros(fraction))
+  }
+}
+
+/**
+ * Use GCD to reduce the fraction to the smallest possible
+ *
+ * From https://stackoverflow.com/a/65927538
+ *
+ * @param fraction
+ */
+function reduce(fraction: Fraction): Fraction {
```

Comment:
I made the comment in the wrong PR, as I saw the changes there
https://github.com/cowprotocol/cowswap/pull/5327/files#r1925818123

Reply:
Thanks, addressed on 40494100dfb6